docker-compose.local.yml

```diff
@@ -21,7 +21,10 @@ services:
 
   mock:
     platform: ${LOCAL_PLATFORM}
+    image: null # This is necessary to override the image specified in the base file
     build: https://github.com/ministryofjustice/hmcts-common-platform-mock-api.git#main
     pull_policy: always
     volumes:
       - ./tmp/logs/mock:/usr/src/app/log
+      - ../hmcts-common-platform-mock-api/lib/tasks:/usr/src/app/lib/tasks # to override the default seed data when developing
+    command: sh -c "bundle exec rails db:create db:migrate db:seed; bundle exec rails mock:demodata:unload; bundle exec rails mock:demodata:load; bundle exec puma -C config/puma.rb"
```

Comment:
This is a duplication from docker-compose.yml,
I'd rather avoid it, but it does not run the main one if I don't add it (maybe because of image: null, not sure)